apple/RNGestureHandlerModule.mm

```diff
@@ -123,55 +125,55 @@ - (void)setBridge:(RCTBridge *)bridge
 #ifdef RCT_NEW_ARCH_ENABLED
 RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(install)
 {
-  RCTCxxBridge *cxxBridge = (RCTCxxBridge *)self.bridge;
-  auto runtime = (jsi::Runtime *)cxxBridge.runtime;
-  decorateRuntime(*runtime);
-  return @true;
+  [self.bridge
+      dispatchBlock:^{
+        RCTCxxBridge *cxxBridge = (RCTCxxBridge *)self.bridge;
+        auto runtime = (jsi::Runtime *)cxxBridge.runtime;
+        decorateRuntime(*runtime);
+      }
+              queue:RCTJSThread];
+
+  return nil;
```

Comment:
You should return `true`  here since I need this information for bridgeless mode.

Reply:
Done